src/Symfony/Component/Console/Command/Command.php

```diff
@@ -164,9 +168,6 @@ public function isEnabled(): bool
      */
     protected function configure()
     {
-        if (!$this->code && \is_callable($this)) {
-            $this->code = new InvokableCommand($this, $this(...));
-        }
```

Comment:
Moved to the constructor to allow overriding the `configure()` method without requiring a call to the parent implementation